packages/space-opera/src/components/mobile_view/mobile_view.ts

```diff
@@ -182,23 +221,51 @@ export class MobileView extends ConnectedLitElement {
           min-field-of-view=${ifDefined(config.minFov)}
           max-field-of-view=${ifDefined(config.maxFov)}
           animation-name=${ifDefined(config.animationName)}
+          @load=${this.newModel}
         >${childElements}</model-viewer>
       </div>
     </div>`;
   }
 
+  /**
+   * Determine the mobile operating system.
+   * This function returns one of 'iOS', 'Android', 'Windows Phone', or
```

Comment:
Haha, Windows Phone! Are any of these even left? I still remember Ballmer screaming about how awesome it was and how it would destroy the iPhone...

Reply:
developers developers developers!!